backend/StreamingServer/middleware.py

```diff
@@ -12,7 +12,7 @@ def __call__(self, request):
             user_token = request.headers.get('Authorization')
         elif request.method == 'POST':
             try:
-                user_token = json.loads(request.body.decode()).get('headers', {}).get('Authorization')
+                user_token = json.loads(request.body.decode("utf-8", "ignore")).get('headers', {}).get('Authorization')
```

Comment:
I know we talked about that a lot, and we kind of always forget what were the problems and their solutions.
Would it be possible to add some comments here to keep track of these things?